spec/amber/cli/commands/generate/error_template_spec.cr

```diff
@@ -0,0 +1,35 @@
+require "../../../../spec_helper"
+require "../../../../support/fixtures/cli/generators/error/*"
+
+module Amber::CLI
+  describe MainCommand::Generate do
+    begin
+      describe "amber generate error" do
+        context "in an `amber new` app with default options" do
+          ENV["AMBER_ENV"] = "test"
+          MainCommand.run ["new", TESTING_APP]
+          Dir.cd(TESTING_APP)
+          MainCommand.run %w(generate error a)
+
+          it "generates expected controller class" do
+            expected = CLIFixtures::ErrorControllerFixture.expected_error_controller
+            File.read("./src/controllers/error_controller.cr").should eq expected
+          end
+
+          it "generates a controller spec file" do
+            File.exists?("./spec/controllers/error_controller_spec.cr").should be_true
+          end
```

Comment:
I didn't test the contents of the spec file, would that be valuable?

Reply:
I think verifying the file exists is fine.  It will become brittle if we check for contents IMO.

Reply:
This is fine, since it should be captured by the `master build spec` if the contents are valid